.gitignore

```diff
@@ -0,0 +1,19 @@
+*.egg-info/
+.eggs/
+*.o
+*.py[co]
+*.so
+*.DS_Store
+_trial_temp*/
+build/
+dropin.cache
+doc/
+docs/_build/
+dist/
+venv/
```

Comment:
If you make a dir called `venv` personally, then you should probably put something in your git global-ignore; we should only put stuff in here that the project itself, or specific recommended tooling (tox, trial, etc) creates.